ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorAliasTokensDemoController.swift

```diff
@@ -0,0 +1,323 @@
+
+        return cell
+    }
+
+    private func textColor(for token: AliasTokens.ColorsTokens) -> UIColor {
+        let black = ColorValue(0x000000)
+        let white = ColorValue(0xFFFFFF)
+
+        switch token {
+        case .background1, .background1Pressed, .background1Selected, .background2, .background2Pressed, .background2Selected, .background3, .background3Pressed, .background3Selected, .background4, .background4Pressed, .background4Selected, .background5, .background5Pressed, .background5Selected, .background5BrandTint, .background6, .background6Pressed, .background6Selected, .backgroundDisabled, .brandBackground4, .background5BrandTintSelected, .brandBackgroundDisabled, .canvasBackground, .stencil1, .stencil2, .foregroundDisabled2, .foregroundOnColor, .brandForeground2, .stroke1, .stroke2, .strokeDisabled, .strokeFocus1, .foregroundDisabled1:
+            return UIColor(dynamicColor: DynamicColor(light: black, dark: white))
+        case .brandBackground3Pressed, .foreground1, .foreground2, .foreground3, .foregroundInverted2, .brandForegroundInverted, .strokeFocus2, .brandBackground1Pressed, .brandForeground1Pressed, .brandStroke1Pressed, .brandStroke1, .brandStroke1Selected:
+            return UIColor(dynamicColor: DynamicColor(light: white, dark: black))
+        case .brandBackground3Selected, .brandBackgroundInverted, .brandBackgroundInvertedDisabled, .foregroundInverted1, .brandForeground3:
+            return .black
+        case .foregroundContrast, .brandForeground1, .brandForeground1Selected, .brandForeground4, .brandForeground5, .brandForegroundDisabled, .background5SelectedBrandFilled, .backgroundInverted, .brandBackground1, .brandBackground1Selected, .brandBackground2, .brandBackground2Pressed, .brandBackground2Selected, .brandBackground3, .strokeAccessible, .background5BrandFilledSelected:
+            return .white
+        }
+    }
+
+    private var aliasTokens: AliasTokens {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return AliasTokens()
+        }
+        return fluentTheme.aliasTokens
+    }
+
+}
+
+private enum ColorAliasTokensDemoSection: CaseIterable {
+    case neutralBackgrounds
+    case brandBackgrounds
+    case neutralForegrounds
+    case brandForegrounds
+    case neutralStrokes
+    case brandStrokes
+
+    var title: String {
+        switch self {
+        case .neutralBackgrounds:
+            return "Neutral Backgrounds"
+        case .brandBackgrounds:
+            return "Brand Backgrounds"
+        case .neutralForegrounds:
+            return "Neutral Foregrounds"
+        case .brandForegrounds:
+            return "Brand Foregrounds"
+        case .neutralStrokes:
+            return "Neutral Strokes"
+        case .brandStrokes:
+            return "Brand Strokes"
+        }
+    }
+
+    var rows: [AliasTokens.ColorsTokens] {
+        switch self {
+        case .neutralBackgrounds:
+            return [.background1,
```

Comment:
How can we guarantee that we'll be able to keep this array in sync when new alias tokens are added?

Reply:
I have thought about that quite a bit and unless we get to separate the colors in different enums in `AliasTokens`, the simplest way would be to create a new enum in `AliasTokens` representing the different categories of each color that we can use to map each color to the right section in the DemoController.